tsconfig.lint.json

```diff
@@ -1,4 +0,0 @@
-{
-  "extends": "./tsconfig",
-  "include": [".eslintrc.js", "**/*.ts", "**/*.js", "**/*.mjs"]
-}
```

Comment:
Please use this for lint and include all js files. We can disable strict flags for lint if required